models/restaurantgroup.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const {
+  Model, Sequelize
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class RestaurantGroup extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  RestaurantGroup.init({
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: Sequelize.UUIDV4,
+      unique: true
+    }
+  }, {
+    sequelize,
+    modelName: 'RestaurantGroup',
+  });
+
```

Comment:
Could we write this part with `sequelize.define`? Sorry but it'll be confusing to have two different ways of defining models in the code base...